app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -94,12 +99,19 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         setupThemeAndBrowsingMode(getModeFromIntentOrLastKnown(intent))
         setContentView(R.layout.activity_home)
         Performance.instrumentColdStartupToHomescreenTime(this)
-        setupToolbarAndNavigation()
 
         if (intent.getBooleanExtra(EXTRA_FINISH_ONBOARDING, false)) {
             FenixOnboarding(this).finish()
         }
 
+        externalSourceIntentProcessors.any {
```

Comment:
nit: can this be put back on one-line? It's especially important if it's in a busy method like `onCreate`.